app.py

```diff
@@ -36,6 +36,30 @@ def list_bibliography_from_db(db, count):
     ).limit(count)
     return bibliography_sets.all()
 
+
+@app.route("/document_matrix")
+@requires_db
+async def format_term_document_matrix(db, request):
+    to_return = [{
+        "bibliography_eid": document.bibliography_eid,
+        "bibliography_options": document.bibliography_options,
+        "processing_options": document.processing_options,
+        "term_list_path": document.term_list_path,
+        "matrix_path": document.matrix_path
+    } for document in list_term_document_matrix_from_db(db, count=10)]
+    return json(to_return)
+
+
+def list_term_document_matrix_from_db(db, count):
```

Comment:
We could use plurals here, `list_term_document_matrices....` the singular is just for the the HTTP API `GET /matrix`.